### PR TITLE
remove-other-tokens handler skript

### PR DESCRIPTION
**Pull request by laclaro**
Remove other tokens for the user, e.g. on /token/init.

This is an event handler script.

### CODE REVIEW

File: remove-other-tokens.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#!/opt/privacyidea/bin/python
+
+from privacyidea.lib.token import get_tokens, get_one_token, remove_token
+from privacyidea.lib.user import User
+import argparse
+from privacyidea.app import create_app
+import logging
+
+__doc__ = """
+This scripts create new tokens for a given user.
```
Comment (cornelinux) [inline remove-other-tokens.py]: I think the short description is wrong. ;-)
The script actually *removes* all other tokens, but the one that was just created.

File: remove-other-tokens.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+"""
+
+# "type": remove other tokens that share the type of the token in the request
```
Comment (cornelinux) [inline remove-other-tokens.py]: Would *all* tokens of this type be removed? Or only the tokens of this very user with the given type?
Please adapt the description a bit.

**Review by cornelinux — CHANGES_REQUESTED** [review]
Please simply clarify the description. Thanks.

File: remove-other-tokens.py (hunk 2 above)
Comment (laclaro) [reply remove-other-tokens.py]: only tokens of the given user, except the one given by serial will be removed.

File: remove-other-tokens.py (hunk 1 above)
Comment (laclaro) [reply remove-other-tokens.py]: You are right, I apparently did miss to edit this line.

**Review by cornelinux — APPROVED** [review]

**Pull request merged**